src/Components/App/App.js

```diff
@@ -74,20 +74,17 @@ class App extends Component {
   }
 
   render() {
-    const peopleArray = [this.state.people]
-
     return (
       <main className="App">
        <h1 className="main-title">SWAPI<span className="main-title-two">BOX</span></h1>
        <ButtonContainer />
-        <Video />
-        {/* <CardContainer
-          people={this.state.people}
-          planets={this.state.planets}
-          vehicles={this.state.vehicles} /> */}
+        {/* <Video /> */}
+
         {/* {
           this.state.scroll.length && <Scroll scrollData={this.state.scroll} />
         } */}
+        <CardContainer
+          vehicles={this.state.cleanedVehicles} />
```

Comment:
This is looking a lot cleaner than when we were sending all the data as a prop to cardContainer.